webpack/assets/javascripts/react_app/redux/actions/notifications/index.js

```diff
@@ -11,7 +11,7 @@ import {
 } from '../../../common/sessionStorage';
 import API from '../../../API';
 import { isNil } from 'lodash';
-const defaultNotificationsPollingInterval = 10000;
+const defaultNotificationsPollingInterval = 100000000;
```

Comment:
Although I feel the need for this change: This is not the place. Please remove this.